include/boost/geometry/io/wkt/write.hpp

```diff
@@ -62,22 +63,67 @@ namespace boost { namespace geometry
 namespace detail { namespace wkt
 {
 
+template <typename Output>
+struct output_formatter
+{
+    template <typename T>
+    static void append(Output& out, T const& v)
+    {
+        out << v;
+    }
+};
+
+template <typename CharT, typename Traits, typename Allocator>
+struct output_formatter<std::basic_string<CharT, Traits, Allocator>>
+{
+    template <typename T>
+    static void append(std::basic_string<CharT, Traits, Allocator>& out, T const& val)
+    {
+        std::basic_stringstream<CharT, Traits, Allocator> ss;
```

Comment:
Prior to C++11 `basic_stringstream` didn't have `Allocator` parameter [1] so please remove it. Plus it's probably not needed to use a custom allocator here.

[1] https://en.cppreference.com/w/cpp/io/basic_stringstream